src/gluonts/transform.py

```diff
@@ -155,6 +155,42 @@ def __call__(self, ts: np.ndarray, a: int, b: int) -> np.ndarray:
         return np.array([b])
 
 
+class ContinuousTimePointSampler:
```

Comment:
This abstraction looks very similar to the `InstanceSampler`: https://github.com/awslabs/gluon-ts/blob/a97518d436dbb8e2cb6e2958d4ab899e5fa43a9d/src/gluonts/transform.py#L94

The only difference seems the `num_instances` constructor argument here. I'm wondering if the two could be condensed somehow?

Reply:
So in continuous time you sample between points in time (float), as opposed to indices (int).